src/main/java/org/apache/solr/benchmarks/task/AbstractTask.java

```diff
@@ -12,18 +12,18 @@
  */
 public abstract class AbstractTask<T> implements Task<T> {
   protected final TaskByClass taskSpec;
-  private final boolean isFiniteTask;
+  private final Long maxExecution;
```

Comment:
It's kind of unclear what `maxExecution` means. At first glance, it seems like it might be a timeout in seconds, whereas it's really the value passed to `ControlledExecutor` that limits the number of executions. I know that this is just following the naming convention of `ControlledExecutor` but maybe we could throw in a comment here as well.